src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  const add = 'addProperties';
+  const remove = 'removeProperties';
+  const clearing = 'clear';
+
+  for (const ch of actions) {
```

Comment:
Destructure `action`

```suggestion
  for (const action of actions) {
```